### ISSUE TITLE
Addresses

### ISSUE THREAD

**Issue by lmatteis** [lmatteis/peer-tweet#9]
If you're using PeerTweet you can share your address using this page. To kick things off here's my address @`2LuvbFkprVtmrkTjq3Ch1hNrBcse`

**Comment by cryptid11** [comment]
TFRtABF4gATQ12mWSfBQ11DJw9U

**Comment by stephanepechard** [comment]
If anyone sees this: @zu1w88SX2ybEdbBjDHE4ShFLkAY

**Comment by arnauldb** [comment]
@34a2YTxiDSiZavf9obHsh83YtfBa

**Comment by stephanepechard** [comment]
Can't find any people.. is it working?

**Comment by arnauldb** [comment]
"Can't find any people.. is it working?" can't find either...I tried to find you stephanepechard, but doesn't seem to work.

**Comment by lmatteis** [comment]
@arnauldb I found your "hello". The system works such that it can only find someone's tweets, if they were broadcast by them in the last 2 hours. @stephanepechard I can't seem to find yours, perhaps you need to re-broadcast by hitting the bottom-left buttons.

**Comment by stephanepechard** [comment]
@lmatteis you seem to be the only one who can see other people timelines, do you have a magic trick?
Btw, I re-broadcast my timeline.

**Comment by arnauldb** [comment]
@lmatteis ok, thank you explaining and sorry for my previous comment stating it wasn't working. So the one we wish to follow has to tweet something first...am I right ?

**Comment by lmatteis** [comment]
@stephanepechard try searching for `2LuvbFkprVtmrkTjq3Ch1hNrBcse`. Again, it wasn't appearing earlier because I do not usually broadcast my tweets.

**Comment by stephanepechard** [comment]
Tried again with no luck. I'm behind a corporate proxy, could it be the reason?

**Comment by lmatteis** [comment]
@stephanepechard could be. Go in settings (gear icon on the left sidebar) and click the "Update DHT Array" button. It should show the number of peers you're connected to in the textarea. If it shows 0, it may be that you're behind a strong firewall. Also try restarting the app.

**Comment by stephanepechard** [comment]
It says 0 yes... Will try again later, at home probably. Sorry for polluting this thread...

**Comment by lmatteis** [comment]
@arnauldb yes exactly. One drawback of this distributed alternative is that users need to re-announce their tweets every so often to keep them alive. This is automatically achieved if you keep the app open. But if you go offline for some time (as was the case for me) new users won't be able to reach your tweets. Also, your followers automatically re-announce your tweets when you're offline - but you need to have a big following for that to work in a sustainable way.

**Comment by manv3lez** [comment]
@lmatteis this is pretty awesome. I downloaded it, created an account, and I believe I PeerTweeted you @2LuvbFkprVtmrkTjq3Ch1hNrBcse. 

PeerTweet combined with your [TorrentPeek search](https://github.com/lmatteis/torrent-peek) is pretty cool. There's a very interesting challenge [out there](https://twitter.com/naval/status/828439863674310658) now with your name on it. 

Have you considered incentivizing nodes to broadcast PeerTweets with a certain frequency, similar to what is being described [here](https://twitter.com/ryanxcharles/status/853962750871162881)?
